src/applications/personalization/profile/mocks/script/drupal-vamc-data/mockLocalDSOT.js

```diff
@@ -62,16 +62,16 @@ const requestAndSaveFile = (url, filepath) => {
 
 /**
  * create local drupal vamc data if it doesn't exist or is older than 2 weeks
- *  | See: https://depo-platform-documentation.scrollhelp.site/developer-docs/how-to-opt-in-to-drupal-as-the-source-of-truth-for
- *  | Or related issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/54395
+ *  | See: [Platform Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/how-to-opt-in-to-drupal-as-the-source-of-truth-for)
+ *  | Or [related issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/54395)
  *
  * @returns {boolean}
  */
 const mockLocalDSOT = () => {
   try {
     if (fs.existsSync(pathToSaveData)) {
       debug('🚀 FILE EXISTS... NOT POPULATING');
-      const fileCreationDate = fs.statSync(pathToSaveData).birthtime;
+      const fileCreationDate = fs.statSync(pathToSaveData).mtime;
```

Comment:
This is a minor fix as the birthtime doesn't update when the file is updated, and instead the mtime should be used to track the last modified time. Some [docs on this node.fs property](https://nodejs.org/api/fs.html#fs_stats_mtime)